### PR TITLE
chore: generated CHANGELOG for origin/release/v54.2.0

### PR DESCRIPTION
**Pull request by RitamAgrawal**
### What does this PR do?
Changelog-v54.2.0

### CODE REVIEW

**Review by gbockus-sf — APPROVED** [review]
cool

**Review by sbudhirajadoc — COMMENTED** [review]
made my updates and added Juliet Shackell to the reviewers list.

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
+# 54.2.0 - February 24, 2022
+
+## Added
+
+#### salesforcedx-vscode
+
+- Run the new *Start Function in Container* command to run a Salesforce Function in a Docker container.  The old command, *Start Function* now runs a function locally. ([PR #3838](https://github.com/forcedotcom/salesforcedx-vscode/pull/3838), [PR #3856](https://github.com/forcedotcom/salesforcedx-vscode/pull/3856))
```
Comment (jshackell-sfdc) [inline packages/salesforcedx-vscode/CHANGELOG.md:7]:
```suggestion
- Run the new *Start Function in Container* command to run a Salesforce Function in a Docker container. The old command, *Start Function* now runs a function locally. ([PR #3838](https://github.com/forcedotcom/salesforcedx-vscode/pull/3838), [PR #3856](https://github.com/forcedotcom/salesforcedx-vscode/pull/3856))
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,22 @@
...
+
+## Fixed
+
+#### salesforcedx-vscode
+
+- We resolved the issue with the *Diff Folder Against Org* command being visible from command palette even when an SFDX project wasn’t open. ([PR #3843](https://github.com/forcedotcom/salesforcedx-vscode/pull/3843)).
```
Comment (jshackell-sfdc) [inline packages/salesforcedx-vscode/CHANGELOG.md:13]:
```suggestion
- We resolved the issue with the *Diff Folder Against Org* command being visible from the command palette even when an SFDX project wasn’t open. ([PR #3843](https://github.com/forcedotcom/salesforcedx-vscode/pull/3843)).
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,22 @@
...
+
+#### salesforcedx-vscode-apex
+
+- We removed some unessential apex classes from the Apex LSP jar file. ([PR #3844](https://github.com/forcedotcom/salesforcedx-vscode/pull/3844))
```
Comment (jshackell-sfdc) [inline packages/salesforcedx-vscode/CHANGELOG.md:17]:
```suggestion
- We removed some unessential Apex classes from the Apex LSP .jar file. ([PR #3844](https://github.com/forcedotcom/salesforcedx-vscode/pull/3844))
```

Comment (jshackell-sfdc) [reply packages/salesforcedx-vscode/CHANGELOG.md]: ".jar" is what CCX recommends, although honestly I don't always follow the rule.  I think it looks funky...

https://confluence.internal.salesforce.com/display/DOCTEAM/file+names+and+extensions

**Review by jshackell-sfdc — APPROVED** [review]

**Review by sbudhirajadoc — APPROVED** [review]

**Pull request merged**